src/app/modules/test-step-selector/test-step-selector.component.ts

```diff
@@ -34,7 +36,7 @@ export class TestStepSelectorComponent implements OnInit {
 
   private copyToClipBoard(node: TreeNode) {
     if (!node.children || node.children.length === 0) {
```

Comment:
actually the test whether copying should be done is:
node.type === 'interaction' || node.type === 'macro'
What do you think?